src/Parser.cpp

```diff
@@ -335,6 +342,7 @@ int PacketParser::start_state(err_code_t& err) {
         // OK
         EXPECT_BYTES("OK\r\n", 4);
         processMessageResult(MSG_OK);
+        m_state = FSM_END;
```

Comment:
Only `flush_all` will return `OK\r\n` in libmc. So it's pretty safe to do the change. If `m_state = FSM_END` is not specified, the client will try to read more response buffer by calling `poll` but will got nothing but `0` (poll timeout), and then disconnect all connections. The connections will be reestablished in further requests because of the re-connecting mechanism.